Form/Type/ImageType.php

```diff
@@ -91,12 +91,8 @@ public function configureOptions(OptionsResolver $resolver)
             ->setDefault('cropper_options', ['autoCropArea' => 1])
             ->setDefault('max_width', 320)
             ->setDefault('max_height', 180)
-            ->setDefault('preview_width', function (Options $options) {
-                return $options['max_width'];
-            })
-            ->setDefault('preview_height', function (Options $options) {
-                return $options['max_height'];
-            })
+            ->setDefault('preview_width', '320px')
```

Comment:
Good idea, however I would keep the callback so that the preview dimensions are still based on the main dimensions by default.
```
->setDefault('preview_width', function (Options $options) {
    return sprintf('%dpx', $options['max_width']);
})
->setDefault('preview_height', function (Options $options) {
    return sprintf('%dpx', $options['max_height']);
})
```

Reply:
good idea :+1:  :)